test/e2eTests/tests/registerPatient_test.js

```diff
@@ -0,0 +1,51 @@
+// Test for:
```

Comment:
Another nit picky comment, but tabs/spacing in this file seem a bit off which makes it a slightly confusing to read. Not a big deal, but would be nice to fix.

Reply:
It might be my editor. I'm using VS Code. Will check.

Reply:
Check if it is using the [`.editorconfig`](https://github.com/esaude/esaude-emr-poc/blob/master/.editorconfig)